### PR TITLE
create folder structure and about page switch to tsx

### PR DESCRIPTION
**Pull request by Jula000**

### CODE REVIEW

File: src/app/about/page.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import AboutPage from "../../components/aboutPage/AboutPage";
```
Comment (yevhenii-horbenko) [inline src/app/about/page.tsx]: please fix folder name to AboutPage

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,25 +1,21 @@
...
+import HeroSection from "../components/HomePage/Hero/HeroSection";
+import Header from "../components/common/Header";
+import PhotoGrid from "../components/HomePage/PhotoGrid/PhotoGrid";
+import AboutComponent from "../components/HomePage/About/AboutComponent";
+import ServiceComponent from "../components/HomePage/Service/ServiceComponent";
+import Portfolio from "../components/HomePage/Portfolio/Portfolio";
+import FaqComponent from "../components/HomePage/FAQ/FaqComponent";
 
-export default function Home() {
+export default function home() {
```
Comment (yevhenii-horbenko) [inline src/app/page.tsx]: why did you change the name?

File: src/components/HomePage/About/AboutComponent.tsx [hunk 3]
```diff
@@ -1,9 +1,10 @@
+import React from "react";
 import Image from "next/image";
-import SocialMediaButtons from "./common/SocialMediaButton";
-import Section from "./common/Section";
-import Button from "./common/Button";
+import SocialMediaButtons from "../../common/SocialMediaButtons";
+import Section from "../../common/Section";
+import Button from "../../common/Button";
 
-export default function AboutSection() {
```
Comment (yevhenii-horbenko) [inline src/components/HomePage/About/AboutComponent.tsx]: Please rename the file and the component name to be AboutSection

File: src/components/HomePage/FAQ/FaqComponent.tsx [hunk 4]
```diff
@@ -44,9 +49,9 @@ const faqData = [
   },
 ];
 
-const FaqComponent = () => {
+const FaqComponent: React.FC = () => {
   return (
-    <div className="text-white p-8">
+    <div className="bg-black text-white p-8">
```
Comment (yevhenii-horbenko) [inline src/components/HomePage/FAQ/FaqComponent.tsx]: consider removing bg-black

File: src/components/HomePage/Hero/HeroSection.tsx [hunk 5]
```diff
@@ -1,10 +1,10 @@
 import React from "react";
 import Image from "next/image";
-import imgBg from "../../../public/hero-bg.png";
-import Button from "./common/Button";
-import GlossyButton from "./common/GlossyButton";
+import Button from "../../common/Button";
 
-export default function HeroSection() {
+import imgBg from "../../../../public/hero-bg.png";
```
Comment (yevhenii-horbenko) [inline src/components/HomePage/Hero/HeroSection.tsx]: could you please try to use @public/hero-bg.png 
IIRC it should work

File: src/components/HomePage/Hero/HeroSection.tsx [hunk 6]
```diff
@@ -18,19 +18,31 @@ export default function HeroSection() {
       <div className="flex font-manrope uppercase flex-col items-start font-semibold text-5xl">
         <div className="flex mb-2">
           <span>Let's</span>
-          <GlossyButton />
```
Comment (yevhenii-horbenko) [inline src/components/HomePage/Hero/HeroSection.tsx]: Please remove glossybutton if you dont use it

File: src/components/common/Header.tsx [hunk 7]
```diff
@@ -1,28 +1,29 @@
...
           <NavLink href="/services">Services</NavLink>
         </nav>
 
-        {/* Contact Me Button */}
-        <Link href="/contact">
-          <Button>Contact Me</Button>
+        <Link href="/contact" passHref>
+          <p className="inline-block">
+            <Button>Contact Me</Button>
+          </p>
```
Comment (yevhenii-horbenko) [inline src/components/common/Header.tsx]: You probably don't need p here, at most you need a div

**Review by yevhenii-horbenko — CHANGES_REQUESTED** [review]
good job! molodiec
I have a few suggestions

File: src/app/about/page.tsx [hunk 8]
```diff
@@ -0,0 +1,13 @@
+import AboutPage from "../../components/AboutPage/AboutPage";
```
Comment (yevhenii-horbenko) [inline src/app/about/page.tsx]: Why do you import it like that?
shouldn't be path be like:
import AboutPage from "../../components/AboutPage";?

File: src/app/page.tsx [hunk 9]
```diff
@@ -1,25 +1,21 @@
...
-import PhotoGrid from "./components/PhotoGrid";
-import AboutSection from "./components/AboutSection";
-import ServiceComponent from "./components/ServiceComponent";
-import Portfolio from "./components/Portfolio";
-import FaqComponent from "./components/FaqComponent";
-import Testimonials from "./components/Testimonials";
-import Footer from "./components/Footer";
+import HeroSection from "../components/HomePage/Hero/HeroSection";
+import Header from "../components/common/Header";
+import PhotoGrid from "../components/HomePage/PhotoGrid/PhotoGrid";
```
Comment (yevhenii-horbenko) [inline src/app/page.tsx]: could you please fix this import?

File: src/app/page.tsx [hunk 10]
```diff
@@ -1,25 +1,21 @@
...
+import AboutComponent from "../components/HomePage/About/AboutSection";
+import ServiceComponent from "../components/HomePage/Service/ServiceComponent";
+import Portfolio from "../components/HomePage/Portfolio/Portfolio";
```
Comment (yevhenii-horbenko) [inline src/app/page.tsx]: same

File: src/app/page.tsx [hunk 11]
```diff
@@ -1,25 +1,21 @@
...
+import FaqComponent from "../components/HomePage/FAQ/FaqComponent";
```
Comment (yevhenii-horbenko) [inline src/app/page.tsx]: why dont you just name this FAQSection?

File: src/app/page.tsx [hunk 12]
```diff
@@ -1,25 +1,21 @@
...
 
 export default function Home() {
   return (
     <main className="flex min-h-screen flex-col bg-main-color">
       <Header />
       <HeroSection />
       <PhotoGrid />
-      <AboutSection />
+      <AboutComponent />
```
Comment (yevhenii-horbenko) [inline src/app/page.tsx]: Why did you rename it?

File: src/app/page.tsx [hunk 13]
```diff
@@ -1,25 +1,21 @@
...
       <ServiceComponent />
```
Comment (yevhenii-horbenko) [inline src/app/page.tsx]: Please dont add Component at the end as it's redundant

File: src/components/HomePage/About/AboutSection.tsx [hunk 14]
```diff
@@ -1,9 +1,10 @@
...
 import Image from "next/image";
-import SocialMediaButtons from "./common/SocialMediaButton";
-import Section from "./common/Section";
-import Button from "./common/Button";
+import SocialMediaButtons from "../../common/SocialMediaButtons";
+import Section from "../../common/Section";
+import Button from "../../common/Button";
 
-export default function AboutSection() {
+const AboutComponent: React.FC = () => {
```
Comment (yevhenii-horbenko) [inline src/components/HomePage/About/AboutSection.tsx]: Please rename the variable name to match file name

**Review by yevhenii-horbenko — CHANGES_REQUESTED** [review]

File: src/components/common/ServicePart.tsx [hunk 15]
```diff
@@ -7,7 +7,7 @@ interface ServiceSectionProps {
   highlights: string[];
 }
 
-const ServiceSection: React.FC<ServiceSectionProps> = ({
+const ServicePart: React.FC<ServiceSectionProps> = ({
```
Comment (yevhenii-horbenko) [inline src/components/common/ServicePart.tsx:10]: please rename the interface as well
you forgot to update ServiceSectionProps

Comment (yevhenii-horbenko) [inline src/components/common/ServicePart.tsx]: please rename the interface as well
you forgot to update ServiceSectionProps

File: src/components/HomePage/Service/index.tsx [hunk 16]
```diff
@@ -16,7 +16,7 @@ const title: string = "Events";
 const description: string =
   "Our event photography service is dedicated to capturing the magic of your special occasions. Whether it's a wedding, corporate event, or milestone celebration, we're there to document every heartfelt moment. We blend into the background, ensuring natural and candid shots that reflect the emotions of the day.";
 
-const ServiceComponent: React.FC = () => {
+const ServiceSection: React.FC = () => {
```
Comment (yevhenii-horbenko) [inline src/components/HomePage/Service/index.tsx]: please update the name to match folder name

File: src/components/common/ServicePart.tsx (hunk 15 above)
Comment (yevhenii-horbenko) [reply src/components/common/ServicePart.tsx]: I guess you can name this file ServiceHighlights
Tbh servicepart name is a bit confusing

**Review by yevhenii-horbenko — CHANGES_REQUESTED** [review]

File: src/app/page.tsx [hunk 17]
```diff
@@ -1,12 +1,10 @@
...
-import FaqComponent from "./components/FaqComponent";
-import Testimonials from "./components/Testimonials";
-import Footer from "./components/Footer";
+import HeroSection from "../components/HomePage/HeroSection";
+import Header from "../components/common/Header";
+import PhotoGrid from "../components/HomePage/PhotoGrid";
+import AboutSection from "../components/HomePage/AboutSection";
+import ServiceSection from "../components/HomePage/ServiceSection";
+import Portfolio from "../components/HomePage/Portfolio";
+import FaqSection from "../components/HomePage/FaqSection";
```
Comment (yevhenii-horbenko) [inline src/app/page.tsx:7]: beautiful

Comment (yevhenii-horbenko) [reply src/app/page.tsx:7]: luv it

File: src/components/HomePage/ServiceSection/index.tsx [hunk 18]
```diff
@@ -1,21 +1,22 @@
+import React from "react";
 import Image from "next/image";
 import { FaArrowRight, FaArrowLeft } from "react-icons/fa";
-import Button from "./common/Button";
-import ServiceSection from "./common/ServiceSection";
+import Button from "../../common/Button";
+import ServicePart from "../../common/ServiceHighlights";
```
Comment (yevhenii-horbenko) [inline src/components/HomePage/ServiceSection/index.tsx]: name doesn't match the file

**Review by yevhenii-horbenko — CHANGES_REQUESTED** [review]
the last one

File: src/components/HomePage/ServiceSection/index.tsx (hunk 18 above)
Comment (Jula000) [reply src/components/HomePage/ServiceSection/index.tsx]: +

**Review by yevhenii-horbenko — APPROVED** [review]
keep it up! you inspire me!

**Pull request merged**